packages/config/src/config/routes.ts

```diff
@@ -72,7 +73,17 @@ export default function getRoutes() {
     {
       method: 'GET',
       path: '/config',
-      handler: applicationConfigHandler,
+      handler: configHandler,
+      config: {
+        auth: false,
+        tags: ['api'],
+        description: 'Retrieve all configuration'
+      }
+    },
+    {
+      method: 'GET',
+      path: '/applicationConfig',
```

Comment:
If this handler is only used by the login app, rename this handler and route to loginConfig, applicationConfig is misleading

Reply:
Changed it as you suggests.